index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Evans_Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="navigation">
+      <div class="logo">
+        <h3>Evans</h3>
+      </div>
+      <nav class="navlinks">
+        <ul class="links">
+          <li class="link"><a href="#portfolio">Portfolio</a></li>
+          <li class="link"><a href="#about">About</a></li>
+          <li class="link"><a href="#contact">Contact</a></li>
+          <li>
+            <a href="#"><i class="fa-regular fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+
+    <section id="hero">
+      <div class="hero_text">
+        <h1 class="title">
+          Hey There <br />
+          I'm Evans
+        </h1>
+        <h2 class="subtitle">I'm a Software Developer</h2>
+        <p class="text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
```

Comment:
- [ ] You can also increase the [letter-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing) and [line height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) properties of this paragraph a bit. To ensure you have a pixel-perfect design.

- Your design.
<img width="190" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227177687-4bae687f-f8a5-45ec-82e9-fbc728ad5cc4.png">

- Figma design
<img width="217" alt="right" src="https://user-images.githubusercontent.com/41163571/227010023-9fcfb014-309c-4293-ba46-014f8e1c0000.png">